public/javascripts/feedbackFormPreview.js

```diff
@@ -0,0 +1,61 @@
+function generatePreview() {
```

Comment:
Can we change this file name to feedbackformpreview.js instead ? This is the naming convention we have being using for all view assets, lets stick to it.